docs/src/index.js

```diff
@@ -9,6 +9,8 @@ import ReactPerf from 'react-addons-perf';
 import { render } from 'react-dom';
 import App from 'docs/src/components/App';
 
+require('core-js');
```

Comment:
@joonarahko That's going to introduce friction for onboarding. I believe that alternatives are good enough. Also requiring a polyfill raises a lot more of other issues, now we get a duplication between what's inlined and what it polyfilled.